packages/e2e-tests/specs/editor/various/copy-cut-paste-whole-blocks.test.js

```diff
@@ -92,4 +92,91 @@ describe( 'Copy/cut/paste of whole blocks', () => {
 		await pressKeyWithModifier( 'primary', 'v' );
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'should handle paste events once', async () => {
+		// Add group block with paragraph
+		await insertBlock( 'Group' );
+		await page.click( '.block-editor-button-block-appender' );
+		await page.click( '.editor-block-list-item-paragraph' );
+		await page.keyboard.type( 'P' );
+		await page.keyboard.press( 'ArrowLeft' );
+		await page.keyboard.press( 'ArrowLeft' );
+		// Cut group
+		await pressKeyWithModifier( 'primary', 'x' );
```

Comment:
Generally, after cutting, it's good to test setup with `getEditedPostContent` to make sure that something happened. If we copy/paste and check the content only after that, it's possible that nothing at all has happened.

Reply:
Good call, updated in https://github.com/WordPress/gutenberg/pull/34430/commits/c8114adab54d433695bc12abc39ffbf72a8d65da